design/interactable.behavior.md

```diff
@@ -0,0 +1,39 @@
+#  Interactable Behavior
+
+The interactable behavior allows the user to assign this behavior to an actor so that the actor can be held and
+used.
+
+## Architecture
+
+Create a new behavior that is an `InteractableBehavior` for being able to add to an actor.
+
+``` ts
+/**
+ * Interactable behavior class containing the target behavior actions.
+ */
+export class InteractableBehavior extends TargetBehavior {
```

Comment:
Can I add an `InteractableBehavior` to any actor and it will become grabbable? Same question for `PenBehavior`.

Reply:
I'm curious about pros and cons - I can see value in both.

---
In reply to: [376071123](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/478#discussion_r376071123) [](ancestors = 376071123)

Reply:
Yes.  The inherent part of the tool behavior is that it is grabbable.  When you add that behavior, we would automatically set the `grabbable` field on the actor to true.  We could add that as a property to the `ToolBehavior` if we need that to be configurable, but not sure how useful the tool would be if it wasn't grabbable.